.github/workflows/ci.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
 
     - name: Install alchemtest
       run: |
-        python -m pip install https://github.com/alchemistry/alchemtest/archive/master.zip
+        python -m pip install https://github.com/DrDomenicoMarson/alchemtestDM/archive/refs/heads/master.zip
```

Comment:
```suggestion
        python -m pip install https://github.com/alchemistry/alchemtest/archive/master.zip
```

Reply:
Here I show that the reviewer could change the alchemtest back as part of the review process.